docs/collecting-data/collecting-from-own-applications/google-amp-tracker/google-amp-1-0-0-2/index.md

```diff
@@ -12,8 +12,7 @@ To learn more about analytics for AMP pages see the [amp-analytics](https://www.
 
 Snowplow is natively integrated into the project, so pages optimized with AMP HTML can be tracked in Snowplow by adding the appropriate `amp-analytics` tag to your pages:
 
-```html
```

Comment:
Both `html` and `xml` are listed as supported... https://prismjs.com/#supported-languages

Reply:
Moved to markup as html/xml seem to be aliases of same markup Prism configuration.  When trying to add them specifically they didn't exist.

I thought it better to move these to markup as thats what they are and not to add future confusion when html/xml aren't supported by Docusaurus specifically and can't be added as additional types.  It is confusing.  See error that's generated in comments below.

Reply:
I would keep them as `html` / `xml` just because it’s more descriptive, and may be an advantage if we move to a different highlighter in the future.

Reply:
That's true - will drop out the html/xml changes